packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -98,6 +98,14 @@ export class Vector2 {
         return result;
     }
 
+	/**
+	 * Gets the polar coordinates of the current Vector2
+	 * @returns a new Polar with the polar coordinates
+	 */
+	public toPolar(): Polar{

```

Comment:
A `toPolarToRef(result: Polar)` would also be nice to have, to avoid creating a new `Polar` object each time.